project-base/app/translations/messages.cs.po

```diff
@@ -1115,85 +1226,31 @@ msgid "Nominal"
 msgstr "Nominální"
 
 msgid "Not refundable"
-msgstr ""
+msgstr "Nerefundovatelný"
```

Comment:
It's used for payment transactions... maybe `Nerefundovatelné` will be better

Reply:
Changed